lang/cs/Org.Apache.REEF.IMRU/API/IMRUJobDefinitionBuilder.cs

```diff
@@ -203,6 +211,17 @@ public IMRUJobDefinitionBuilder SetUpdateTaskMemory(int memory)
         }
 
         /// <summary>
+        /// Sets Per Map Configuration
```

Comment:
I think we should generalize this a bit. How about allowing for arbitrary additional `IConfiguration` instances to be added to the Driver? Then, this would just be one special case of that generic mechanism.